tests/io/json_tests.cpp

```diff
@@ -171,7 +171,7 @@ static bool equal( const fc::variant& a, const fc::variant& b )
          return true;
       case fc::variant::type_id::blob_type:
       default:
-         FC_THROW_EXCEPTION( fc::invalid_arg_exception, "Unsupported variant type: " + a.get_type() );
```

Comment:
`char[] + int` doesn't make sense, but `char[] + string` does.

Reply:
Better(?) alternative: use format string and int argument. See https://github.com/bitshares/bitshares-fc/pull/102#pullrequestreview-189925457

Reply:
Huh, yeah that totally works. I just assumed it didn't since it wasn't there to begin with. :]